uat/testing-features/src/main/resources/greengrass/components/recipes/client_ipc.yaml

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+---
+RecipeFormatVersion: '2020-01-25'
+ComponentName: aws.greengrass.client.IpcClient
+ComponentVersion: '1.0.0'
+ComponentDescription: IPC Client for tests
+ComponentPublisher: Amazon
+ComponentConfiguration:
+  DefaultConfiguration:
+    topicsToSubscribe: "*"
+
+Manifests:
+  - Artifacts:
+      - URI: classpath:/greengrass/artifacts/aws.greengrass.client.IpcClient.jar
+        Permission:
+          Read: ALL
+          Execute: ALL
+    Lifecycle:
+      Run: |
+        java -DcomponentName="LocalIpcSubscriber" -jar {artifacts:path}/aws.greengrass.client.IpcClient.jar {configuration:/topicsToSubscribe}
```

Comment:
take arguments into double quotes. 
For example your script will fail when value in {artifacts:path} or {configuration:/topicsToSubscribe} contains space

Must be 
java -DcomponentName="LocalIpcSubscriber" -jar "{artifacts:path}/aws.greengrass.client.IpcClient.jar" "{configuration:/topicsToSubscribe}"

Reply:
fixed 23fa329e42c9d0841d7aff88c9b977db8d93de1e